parachains/runtimes/assets/statemine/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2021 Parity Technologies (UK) Ltd.
+// Copyright (C) 2021-2022 Parity Technologies (UK) Ltd.
```

Comment:
My latest information that the end date is not relevant. Did that legal view change recently?

Reply:
I didn't hear anything, just updating while I was already in these files and spotted them and know we've updated other repos.